tests/TestSupport/test-scripts/test-mlr.json

```diff
@@ -0,0 +1,33 @@
+{
+ "modules": {
+ "PlanetPopulation": " ",
+ "StarCatalog": " ",
+ "OpticalSystem": " ",
+ "ZodiacalLight": " ",
+ "BackgroundSources": " ",
+ "PlanetPhysicalModel": " ",
+ "Observatory": " ",
+ "TimeKeeping": " ",
+ "PostProcessing": " ",
+ "Completeness": " ",
+ "TargetList":" ",
+ "massLuminosityRelationship": "Fernandes2010",
```

Comment:
`massLuminosityRelationship` must be defined at the top level (outside of the modules dictionary)